build/Dockerfile.ci

```diff
@@ -17,8 +17,7 @@ ENV GOTOOLCHAIN=local
 
 ENV GOEXPERIMENT=strictfipsruntime
 
-# dos2unix is needed to build CNI plugins
```

Comment:
is dos2unix no longer needed to build CNI plugins ?

Reply:
At the least it hasnt been required in the konflux builds.

https://github.com/openshift/windows-machine-config-operator/blob/master/Containerfile

Reply:
I follow